rfpremier/ah-django
# #164857209 Setup Coveralls

### What does this PR do?

This PR intergrates coveralls so as to report on the coverage of the tests

### Tasks to be completed

- [x] Install coveralls & coverage
- [x] Configure travis.yml to intergrate coveralls
- [x] Update readme with the  coveralls badge

### How can this be manually tested?

- Clone the repo and checkout to the `ch/164857209-integrate-coveralls` branch
- Install new dependencies via `pip install -r requirements.txt`
- Run `coverage run --source=authors manage.py test` in your terminal to mimic what travis will run
- Run `coverage report` the % you see is what will be displayed by coveralls

### PT stories relevant to this PR?

[#164857209](https://www.pivotaltracker.com/n/projects/2321565/stories/164857209)
